apps/backend/convex/entities/types.ts

```
@@ -21,6 +21,7 @@ export type Thread = Infer<typeof ThreadReturn>
 export type ThreadWithDetails = Infer<typeof ThreadWithDetailsReturn>
 export type User = Infer<typeof UserReturn>
 
+export type MessageRoles = Infer<typeof MessageRoles>
```

Biome

[error] Shouldn't redeclare 'MessageRoles'. Consider to delete it or rename it.

'MessageRoles' is defined here:


(lint/suspicious/noRedeclare)